network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -11,18 +11,16 @@
 
 {% block content %}
 <div class="container">
-  <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center homepage-feature">
-    <div class="col-md-6 mb-3 d-sm-none">
-      {% image page.hero_image fill-1152x770-c100 format-jpeg class="full-bleed-xs" %}
-    </div>
+  <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center flex-sm-row-reverse homepage-feature">
+      <picture class="col-sm-6 mb-3 mb-sm-0">
+          <source srcset="{{page.hero_image}}" media="(min-width: 768px)">
+          {% image page.hero_image max-1080x800 format-jpeg class="full-bleed-xs" %}
```

Comment:
Is there a way for us to specify the `alt` text? I just noticed that if the image can't load it says `twitter-card-en.png` which is not the most useful thing for users to see.

Reply:
Thanks for pointing that out @Pomax —done! =)